cartridges/int_bolt_embedded_sfra/cartridge/controllers/Bolt.js

```diff
@@ -148,4 +149,59 @@ server.post('GetAccount', function (req, res, next) {
     return next();
 });
 
+server.post('CreateCompleteAccount', function (req, res, next) {
```

Comment:
Does this work as an upsert or will you be making a separate change for that? On every Bolt login we'll be calling CreateCompletePlatformAccount so I'd expect the majority of calls, the merchant account should already exist and we're just augmenting what's already there

Reply:
I will make a separate change for the upsert. But then merchant proxy need to pass the platform account id then because email is not unique for sfcc accounts.